federation-2/router-bridge/src/planner.rs

```diff
@@ -651,6 +663,7 @@ impl Default for QueryPlannerConfig {
                 enable_defer: Some(false),
             }),
             graphql_validation: true,
+            reuse_query_fragments: None,
```

Comment:
so the JS code looks for bool value if it is explicitly provided OR defaults to true for undefined, so yeah I think we probably should use `Some(true)` but it would behave the same

Reply:
Default being "I have no opinion, let the JS code do whatever it does if the field is not provided" (some sort of null/undefined situation).

Reply:
@dariuszkuc I'm thinking it makes sense to have exactly one piece of software that has an opinion about what the default value of this flag is (`@apollo/query-planner`, not router-bridge or router) rather than multiple layers each trying to guess what the default should be?

Reply:
ie if there's a release of Federation that's like "2.6: reuseQueryFragments now defaults to false", I don't think you'd want to have to make a change in router-bridge to make that happen?

Reply:
Agreed - if the default changes it would have to change in 2 places. That makes sense to me.

Reply:
👍 to let the `query-planner` decide on the defaults... but do we need to update comment on the field above?